controller/cache/cache.go

```diff
@@ -217,13 +227,13 @@ func (c *liveStateCache) getCluster(server string) (clustercache.ClusterCache, e
 	}
 
 	clusterCache = clustercache.NewClusterCache(cluster.RESTConfig(),
-		clustercache.SetSettings(c.cacheSettings),
+		clustercache.SetSettings(cacheSettings.clusterSettings),
```

Comment:
I'm a little confused by this, I cannot find any related change to the receiver in `SetSettings()`

Reply:
The `cacheSettings.clusterSettings` hold `clustercache.Settings` . This is the same type that `SetSettings` needs. So it does not have to be changed.